src/test/java/lotto/domain/LottoNumberTest.java

```diff
@@ -0,0 +1,23 @@
+package lotto.domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class LottoNumberTest {
+
+    @Test
+    public void 로또숫자_범위가_1이상_45이하_테스트_45초과일_때(){
```

Comment:
`opt + cmd + L`을 이용하여 포맷팅을 진행해보면 어떨까요?

```suggestion
    public void 로또숫자_범위가_1이상_45이하_테스트_45초과일_때() {
```

Reply:
테스트는 포맷팅이 안되었군요  ㅠ 신경써서 진행하겠습니다!!!